tests/es/test_cli_client.py

```diff
@@ -0,0 +1,12 @@
+from connectors.es.cli_client import CLIClient
+
+
+def test_overrides_user_agent_header():
+    config = {
+        "username": "elastic",
+        "password": "changeme",
+        "host": "http://nowhere.com:9200",
+    }
+    cli_client = CLIClient(config)
+
+    assert cli_client.client._headers["user-agent"] == CLIClient.user_agent
```

Comment:
Might want to assert against a string literal, since this test will pass even if the user_agent isn't what you want it to be